cric-predict/src/Components/CreateGroup.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import '../Stylesheets/App.css';
+import axios from 'axios';
+import {Button, Modal, Form, Row, Col}  from 'react-bootstrap';
+
+
+export default class CreateGroupsComponent extends React.Component  {
+
+        constructor(){
+            super();
+            this.state = {
+                name: '',
+                privacy: 'public',
+                all_users:[],
+                selected_users: [],
+                filtered_users:[],
+                users_copy: [],
+                showHide : false
```

Comment:
why not name it `showModal` so it conveys clearly what it's doing.

Reply:
I guess, I may have to show a modal in other components as well later on so I specified that this one is to create group